addons/web/static/src/core/l10n/numbers.js

```diff
@@ -1,9 +1,13 @@
 /** @odoo-module **/
 
-import { escapeRegExp, intersperse, sprintf } from "../utils/strings";
+import { escapeRegExp, intersperse } from "../utils/strings";
 import { localization } from "./localization";
 import { _lt } from "./translation";
 
+function throwInvalidNumberError(value) {
```

Comment:
really? no!
just create 
```js
class InvalidNumberError extends Error {}
```
and simply throw it, no need for an helper

Reply:
I don't want to repeat the message

Reply:
i would throw
```js
class InvalidNumberError extends Error {
    constructor(value) {
        super(...arguments);
        this.message = `"${value}" is not a correct number`;
    }
}
```
?

Reply:
or maybe just repeat
```js
throw new Error(`"${value}" is not a correct number`);
```
and not use a class at all?